src/reporters/ReporterCollectData.ts

```diff
@@ -0,0 +1,44 @@
+import { inject, injectable } from 'inversify';
+import { ReporterUtils } from '.';
+import { ArgumentsProvider } from '../scanner';
+import { Types } from '../types';
+import { PracticeWithContextForReporter, IReporter } from './IReporter';
+import { ProjectComponent } from '../model';
+import Axios from 'axios';
+
+@injectable()
+export class ReporterCollectData implements IReporter {
```

Comment:
All reporters are named `SomethingReporter`. I'd prefer to hold the same standard.